test/connectivity/docker-instance-manager.js

```diff
@@ -0,0 +1,55 @@
+const { execSync } = require('child_process');
+const debug = require('debug')('connectivity/docker-instance-manager');
+const fs = require('fs');
+
+const dockerEnvsPath = '/tmp/compass-connectivity-docker-test-envs';
+const dockerEnvsRepo = 'git@github.com:mongodb-js/devtools-docker-test-envs.git';
+const dockerEnvsRepoCommit = '7297467d208a8bfc15d4f53faf983ba9cbe87360';
+
+const directoryExists = (directoryPath) => {
+  // eslint-disable-next-line no-sync
+  return fs.existsSync(directoryPath) && fs.lstatSync(directoryPath).isDirectory();
+};
+
+const cloneDockerEnvsRepo = () => {
+  if (directoryExists(dockerEnvsPath)) {
+    execSync(`rm -Rf ${dockerEnvsPath}`);
```

Comment:
I always get all tingly when I see this :) I don’t know if anybody will ever try to run this in a path with spaces or other shell-relevant characters, but can we maybe add quotes around the path to make this a little more foolproof (mostly thinking about people who run this test locally here)?

```suggestion
    execSync(`rm -Rf "${dockerEnvsPath}"`);
```

(For the other `execSync()` calls as well, I guess, but `rm -Rf` is the one where it matters :slightly_smiling_face:)

Reply:
Good call - I think that rm rf might not really be needed either, I'll see if we can do it another way just to ensure if someone changes up that `dockerEnvsPath` it doesn't delete something important.